web/src/styles.css

```
@@ -2356,3 +2356,265 @@ button:disabled[aria-busy="true"] {
+}
+/* US3 (T034): the stable aria-live region is visually hidden — it carries the
+   localized announcement for screen readers; the inline .named-picker-status
+   spans handle the visible state text. */
+.named-picker-live {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)